index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week1</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <section class = "header1">
+        <nav>
+            <ul>
+                <li><a href=""> <img src="images/swiggylogo.png" class = "swiggylogo"> </a></li>
+                <li> <a href="" class="setloc"> <strong>Set location</strong> </a> </li>
+                <li class="chooseloc"> <a href=""> Click to Choose Location </a> </li>
+            </ul>
+            <ul>
+                <strong>
+                    <!-- <li><a href=""></a></li> -->
+                    <li><a href=""><strong>Help</strong></a></li>
+                    <li><a href=""><strong>Sign In</strong></a></li>
+                    <li><a href=""><strong>Cart</strong></a></li>
+                </strong>
+            </ul>
+        </nav>
+    </section>
+
+    <section class = "header2">
+        <nav>
+            <ul>
+                <li><a href="">Home</a> <small>/</small></li>
+                <li><a href="">Banglore</a> <small>/</small></li>
+                <li><a href="">Ashok Nagar</a> <small>/</small></li>
+                <li><a href="">Meghana Foods</a></li>
+            </ul>
+        </nav>
+    </section>
+
+    <section class="box1">
+        <img src="images/box1_image.jpeg" alt="">
+        <div>
+            <h1><strong>Punjabi Momos</strong></h1>
+        </div>
+    </section>
+
+    <main>
+        <div id="divleft">
```

Comment:
Don't name classes based on their position as left/right/middle etc.

This naming is fragile and prone to rewrites as the designs keep evolving.

Instead, always give contextual class names like categories/menu-items/cart to represent what the particular section houses. That way even if the section is moved to another position, the class name would still make sense.